### PR TITLE
Add functions for compression stats

### PR DESCRIPTION
**Pull request by gayyappan**
Add chunk_compression_stats and hypertable_compression_stats
functions to get before/after compression sizes

### CODE REVIEW

File: tsl/src/dist_util.c [hunk 1]
```diff
@@ -234,6 +234,23 @@ dist_util_remote_chunk_info(PG_FUNCTION_ARGS)
...
 
+Datum
+dist_util_remote_compressed_chunk_info(PG_FUNCTION_ARGS)
+{
+	char *node_name;
+	StringInfo query_str = makeStringInfo();
+	/* Strict function */
+	Name schema_name = PG_GETARG_NAME(1);
+	Name table_name = PG_GETARG_NAME(2);
+	Assert(!PG_ARGISNULL(0) && !PG_ARGISNULL(1) && !PG_ARGISNULL(2));
```
Comment (svenklemm) [inline tsl/src/dist_util.c:245]: can you make this an actual check strict checks only apply when calling functions from sql so its better to have those checks in place and also PG_NARGS

Comment (gayyappan) [reply tsl/src/dist_util.c:245]: Can you elaborate what you are asking for?

Comment (svenklemm) [reply tsl/src/dist_util.c:245]: change the assert into if and check PG_NARGS == 3

Comment (gayyappan) [reply tsl/src/dist_util.c:245]: I would like to understand the issue better. This is called via SQL/Pgsql and declared as strict. Are you saying that we are missing the strictness check via PG here? Isn't that the case only if we call via C?

Comment (erimatnor) [reply tsl/src/dist_util.c:245]: I think what @svenklemm is saying is that if the function is marked `STRICT` it won't be called if any of the arguments are NULL, so `PG_NARGS` must be equal to 3.

File: tsl/src/dist_util.c [hunk 2]
```diff
@@ -234,6 +234,23 @@ dist_util_remote_chunk_info(PG_FUNCTION_ARGS)
 	return dist_util_remote_srf_query(fcinfo, node_name, query_str->data);
 }
 
+Datum
+dist_util_remote_compressed_chunk_info(PG_FUNCTION_ARGS)
+{
+	char *node_name;
+	StringInfo query_str = makeStringInfo();
```
Comment (svenklemm) [inline tsl/src/dist_util.c:241]:
```suggestion
    if (PG_NARGS() != 3 || PG_ARGISNULL(0) || !PG_ARGISNULL(1) || !PG_ARGISNULL(2))
       PG_RETURN_NULL();
	char *node_name;
	StringInfo query_str = makeStringInfo();
```

**Pull request merged**